subsys/bootloader/cmake/sign.cmake

```diff
@@ -4,11 +4,16 @@
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
 #
 
-set(sign_depends kernel_elf ${KEY_FILE_DEPENDS})
+set(sign_depends
+	${KERNEL_ELF} # Depends on hex/zephyr.elf being built before signing.
+	${SIGN_KEY_FILE_DEPENDS} # Generate keys before signing when needed
+	)
 
-set(hash_file firmware.sha256)
-set(signature_file firmware.signature)
-set(SIGNATURE_PUBLIC_KEY_FILE public.pem)
+set(file_path zephyr/nrf/subsys/bootloader/image)
+
+set(hash_file ${PROJECT_BINARY_DIR}/firmware.sha256)
+set(signature_file ${PROJECT_BINARY_DIR}/firmware.signature)
+set(signature_public_key_file ${PROJECT_BINARY_DIR}/public.pem)
```

Comment:
I guess it's convention that these variables are in CAPS?

Reply:
There isn't really a convention unfortunately.

Reply:
okei changed it to caps anyways in the hopes of making it a convention.
So target = lowercase, set variables = uppercase at least it's what I tried to follow.